src/i18n/langResources/lang.de.ts

```diff
@@ -27,6 +27,7 @@ export const langDe: ILangResource = {
   tsOfficialPresetMessage: 'Offizielles TS Icon wird',
   jsonOfficialPresetMessage: 'Offizielles JSON Icon wird',
   hideFoldersPresetMessage: 'Ordner Icons werden',
+  foldersAllDefaultIconPresetMessage: 'Ordner Icons Auflegemaske werden',
```

Comment:
My guess is that this is going to happen also with many of the remaining languages... so it's maybe a good idea to get rid of the composed messages and have more translations. I think it will be better for everyone to allow us to have the best translations possible.